travis/run_tests.sh

```diff
@@ -12,9 +12,11 @@
 # -----------------------------------------------------------------------------
```

Comment:
Some scripts have this `#set -e` command under bash. Should they all have it or not?

Reply:
`set -e` means it should exit if anything gives an error. I presume there was a reason why I put this in at the time, but I don't remember the specifics now. I don't think it's much of an issue in general whether the scripts have it or not - I'd personally leave it as it is.

Reply:
it is only used intwo scripts: `travis/run_tests.sh` and `build/build_atchem2.sh`. But in the first script it is with a `#` and in the second without. Which one is correct?

Reply:
You're absolutely right. The `#` renders this one useless anyway, it's just a comment and can be removed. Let's keep the one in `build_atchem2.sh`.